hawkular-alerts-actions/hawkular-alerts-actions-impl/src/main/java/org/hawkular/alerts/actions/standalone/ServiceNames.java

```diff
@@ -33,20 +35,24 @@
 
     private static String JNDI_ACTIONS_METRICS = "java:global/hawkular-metrics/hawkular-alerts/CassActionsServiceImpl";
     private static String JNDI_ACTIONS_STANDALONE = "java:global/hawkular-alerts/CassActionsServiceImpl";
-    private static String JNDI_DEFINITIONS_METRICS =
-            "java:global/hawkular-metrics/hawkular-alerts/CassDefinitionsServiceImpl";
+    private static String JNDI_DEFINITIONS_METRICS = "java:global/hawkular-metrics/hawkular-alerts/CassDefinitionsServiceImpl";
     private static String JNDI_DEFINITIONS_STANDALONE = "java:global/hawkular-alerts/CassDefinitionsServiceImpl";
 
     public enum Service {
-        ACTIONS_SERVICE,
-        DEFINITIONS_SERVICE
+        ACTIONS_SERVICE, DEFINITIONS_SERVICE
     }
 
     private static Map<Service, String> services;
 
     static {
         services = new HashMap<>();
-        if (System.getProperty(HAWKULAR_ALERTS_ACTIONS_ENV, STANDALONE).equals(STANDALONE)) {
+        String env = null;
+        try {
+            env = (String) new InitialContext().lookup("java:comp/env/" + HAWKULAR_ALERTS_ACTIONS_ENV);
```

Comment:
Right, because 'standalone' was being used as the default for the getProperty, it masked the fact that it was not actually resolving the property.  When I tried to run in Hservices it didn't work and I discovered the default-value issue.  When I fixed that I then found it didn't work in standalone...